HypVINN/utils/visualization_utils.py

```diff
@@ -161,10 +161,11 @@ def plot_qc_images(save_dir,orig_path, prediction_path, padd=45, lut_file=HYPVIN
 
     image = nib.as_closest_canonical(nib.load(orig_path))
     pred  = nib.as_closest_canonical(nib.load(prediction_path))
+    pred_arr = hypo_map_subseg_2_fsseg(pred.get_fdata(),reverse=True)
```

Comment:
Instead of IMG.get_fdata(), you may want to use np.asarray(omg.datsobj) as the former always returns float arrays, but here you are loading a segmentation, i.e. int data.